src/Illuminate/Support/Collection.php

```diff
@@ -679,7 +679,7 @@ public function first(callable $callback = null, $default = null)
      * @param  mixed  $value
      * @return mixed
      */
-    public function firstWhere($key, $operator, $value = null)
+    public function firstWhere($key, $operator = null, $value = null)
```

Comment:
In your PR text above, you give an example of this being `$collection->firstWhere('published', true)` - but here the `$operator` is `null`? Thats not the same thing? Does it work as expected?

Reply:
Yes, because both `firstWhere` and `where` call the `operatorForWhere` function which, when only passed 1 argument will fallback to `$operator = '='` and `$value = true`